apps/web/app/app.css

```diff
@@ -0,0 +1,44 @@
+@layer reset, base, tokens, recipes, utilities;
```

Comment:
Would it be a pain to move this into `packages/ui` eventually? Then we could also load it from there for the `extension`